packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -108,7 +108,9 @@ const useRootBaseClassName = makeResetStyles({
   ...createCustomFocusIndicatorStyle({
     borderColor: tokens.colorTransparentStroke,
     borderRadius: tokens.borderRadiusMedium,
-    outline: `${tokens.strokeWidthThick} solid ${tokens.colorTransparentStroke}`,
+    outlineColor: tokens.colorTransparentStroke,
```

Comment:
you can just replace this with

```ts
...shorthands.outline(width, style, color)
```

Reply:
at some point in Griffel, we started stripping all shorthands, which is why using `outline`does not work. I guess we never noticed the effect of that in Fluent until we tested in HC again.

Reply:
@ling1726 this call is under `makeResetStyles` that **accepts** shorthands.

@smhigley can you please check if this change is necessary at all?

Reply:
oh yeah I forgot it was using `makeResetStyles` now, then yeah I can't imagine why this change is necessary either

Reply:
this change isn't necessary for functionality, I just made it for consistency with the other places we write the same styles.

Reply:
@smhigley here it will emit bigger CSS (as we use `makeResetStyles` that produces a single CSS rule), so I suggest to revert it back.